### PR TITLE
Add Livewire plugin

### PR DESCRIPTION
**Pull request by owenvoke**

### CODE REVIEW

File: src/Plugin.php [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
- */
-final class Plugin
-{
-    // ..
+use Livewire\Testing\TestableLivewire;
+use Pest\Livewire\InteractsWithLivewire;
+
+Pest\Plugin::uses(InteractsWithLivewire::class);
+
+if (!function_exists('livewire')) {
```
Comment (nunomaduro) [inline src/Plugin.php]: I think this function don't makes sense as livewire already gives static access to test liveware components.

File: tests/Plugin.php [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <?php
 
-use Pest\PluginName\Plugin;
+it('has global function', function () {
+    assertTrue(function_exists('livewire'));
+});
 
-it('has plugin', function () {
-    assertTrue(class_exists(Plugin::class));
+it('has plugin trait method', function () {
```
Comment (nunomaduro) [inline tests/Plugin.php]: You should call the method using the variants that the mthod have an assert a few things.

File: tests/Plugin.php [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <?php
 
-use Pest\PluginName\Plugin;
+it('has global function', function () {
```
Comment (nunomaduro) [inline tests/Plugin.php]: This can be removed.

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: tests/Plugin.php (hunk 2 above)
Comment (owenvoke) [reply tests/Plugin.php]: Just working on this. 👍

**Review by octoper — APPROVED** [review]

**Review by nunomaduro — APPROVED** [review]

**Pull request merged**